### PR TITLE
Add additional ways to contribute part to update zh doc

### PR DESCRIPTION
**Pull request by gaoguangze111**
Add 'additional ways to contribute' part in Chinese to update zh docutation.

### CODE REVIEW

File: content/zh/docs/contribute/_index.md [hunk 1]
```diff
@@ -170,4 +170,10 @@ documentation, but it should help you get started.
   - 对文档测试提出改进建议
   - 对 Kubernetes 网站或其他工具提出改进建议
 
+
```
Comment (tanjunchen) [inline content/zh/docs/contribute/_index.md]: advice:don't delete origin english
it is easy to review to  reviewer.

Comment (gaoguangze111) [reply content/zh/docs/contribute/_index.md]: OK, thx

File: content/zh/docs/contribute/_index.md [hunk 2]
```diff
@@ -170,4 +170,22 @@ documentation, but it should help you get started.
...
+- To contribute to the Kubernetes community through online forums like Twitter or Stack Overflow, or learn about local meetups and Kubernetes events, visit the [Kubernetes community site](/community/).
+-->
+
+- 如果您要通过 Twitter 或 Stack Overflow 等在线论坛为社区做贡献，或了解本地会议及Kubernetes事件，请查看 [Kubernetes community site](/community/).
+
+<!--
+- To contribute to feature development, read the [contributor cheatsheet](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet) to get started.
+-->
+
+- 如果您要开发新的特性，请阅读 [contributor cheatsheet](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet).
```
Comment (tanjunchen) [inline content/zh/docs/contribute/_index.md]: contributor cheatsheet 
贡献者指南

File: content/zh/docs/contribute/_index.md [hunk 3]
```diff
@@ -170,4 +170,22 @@ documentation, but it should help you get started.
...
+## Additional ways to contribute
+-->
+
+## 其他贡献途径
+
+<!--
+- To contribute to the Kubernetes community through online forums like Twitter or Stack Overflow, or learn about local meetups and Kubernetes events, visit the [Kubernetes community site](/community/).
+-->
+
+- 如果您要通过 Twitter 或 Stack Overflow 等在线论坛为社区做贡献，或了解本地会议及Kubernetes事件，请查看 [Kubernetes community site](/community/).
```
Comment (tanjunchen) [inline content/zh/docs/contribute/_index.md]:
```suggestion
- 如果您要通过 Twitter 或 Stack Overflow 等在线论坛为社区做贡献，或了解本地会议及 Kubernetes 事件，请查看 [Kubernetes 社区网站](/community/).
```

**Pull request merged**